util/topgen/intermodule.py

```diff
@@ -102,10 +102,16 @@ def elab_intermodule(topcfg: OrderedDict):
             # Add to definitions
             if "package" in req_struct:
                 package = req_struct["package"]
-            else:
-                assert "package" in rsp_struct, "Either req/rsp shall have 'package' field"
+            elif "package" in rsp_struct:
                 package = rsp_struct["package"]
+            else:
+                package = ""
 
+            # Assume it is logic
+            # req_rsp doesn't allow logic
```

Comment:
sorry i think i'm missing something..but what's the reason response should not be logic..?

Reply:
Oh.. It is not due to something cannot be implemented but more like personal preferences. If req/rsp need to be pair, I thought that it normally comes and goes with some data, so structures may be necessary. If something like `init_req` and `init_done` thing, then two logic connection may be fine. Do you think logic also should be supported in req_rsp type too?

Reply:
ah that makes sense.

That sounds good to me.